### ISSUE TITLE
Pi3, bluetooth and other extensions

### ISSUE THREAD

**Issue by psychogenic** [me-no-dev/RasPiArduino#84]
Greets,

I've got an Arduino SDK-based project we're looking to move to the Raspberry Pi 3.

This project seems like a perfect fit to allow porting the existing codebase, but it seems likely I'll need to add in a good deal of functionality, namely to get working with the bluetooth interface.

My questions:
1) Would such extensions be of use/interest to you?
2) I'll probably base the BLE API on something existing (e.g. the nRF BLE libs created for the BLE Nano2)--do you have a preference in this regard?
3) Do you have any information, preferences, style guidelines or anything that would make merging my eventual enhancements/pull requests easier?

Thanks!
Pat Deegan

**Comment by me-no-dev** [comment]
Yes surely I will appreciate any help with the code :) when I started writing it, raspberries did not have BT :D
